sorc/chgres_cube.fd/input_data.F90

```diff
@@ -537,6 +546,42 @@ subroutine read_input_sfc_data(localpet)
                                    ungriddedUBound=(/lsoil_input/), rc=rc)
  if(ESMF_logFoundError(rcToCheck=rc,msg=ESMF_LOGERR_PASSTHRU,line=__LINE__,file=__FILE__)) &
     call error_handler("IN FieldCreate", rc)
+    
+ 
+
+ if (.not. vgfrc_from_climo) then
+ print*,"- CALL FieldCreate FOR INPUT VEGETATION GREENNESS."
```

Comment:
Be sure to add these new fields to 'cleanup_input_sfc_data'.

Reply:
They've been added. I also added an additional field to the corresponding routine for target fields for now. It's related to the terrain height adjustment, so it may be removed depending on how we decided to proceed with that flag.